### PR TITLE
mv README.md,CHANGELOG.md to docs

### PR DESCRIPTION
**Pull request by hhio618**
### Refs
#268

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1 @@
...
-    If you are building a competing client, please contact us. The miner specifications are off-chain and the validity of the mining process hinges on the consensus of the Tellor community to determine what proper values are. Competing clients that change different pieces run the risk of being disputed by the commmunity.
-
-    There is no guaruntee of profit from mining.
-
-    There is no promise that Tellor Tributes currently hold or will ever hold any value.
-
-
-### Copyright
-Tellor Inc. 2019
+See [docs](https://docs.tellor.io/miner-documentation/miner)
```
Comment (krasi-georgiev) [inline README.md:1]: I am thinking to add 2 links here like
```suggestion
The project uses Gitbook linked to the [docs](./docs) folder of the repo.
[Telliot Docs](https://docs.telliot.tellor.io)
[Tellor Project](https://docs.tellor.io)

```

Comment (krasi-georgiev) [reply README.md]: I added this just an example, can you please add a bit better formating.

https://github.com/hhio618/telliot/tree/move-md-to-docs

Also I just noticed that the links to the assets also needs updating
https://github.com/hhio618/telliot/tree/move-md-to-docs/docs
The images are broken but probably best to fix these with the refactoring of the Readme page anyway.

Comment (hhio618) [reply README.md]: Ok sure.
I'll fix all broken links in a separate PR.

**Review by krasi-georgiev — APPROVED** [review]

**Comment by krasi-georgiev** [comment]
Please fix the failing test and will merge. Thanks!

**Comment by hhio618** [comment]
You're welcome, done!

**Pull request merged**